unbreak https://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git/commit/copy-firmware.sh?id=3e79f6b83bd0d193bbf068b3c8da570f39acc6bb
a nice optimisation, but we don't have this realpath..
(or rdfind)
--- a/copy-firmware.sh
+++ b/copy-firmware.sh
@@ -119,12 +119,6 @@
     fi
 done
 
-$verbose rdfind -makesymlinks true "$destdir" -outputname $rdfind_results "$quiet"
-find "$destdir" -type l | while read -r l; do
-    target="$(realpath "$l")"
-    ln -fs "$(realpath --relative-to="$(dirname "$(realpath -s "$l")")" "$target")" "$l"
-done
-
 exit 0
 
 # vim: et sw=4 sts=4 ts=4
